libs/blocks/global-navigation/global-navigation.js

```diff
@@ -204,9 +204,28 @@ const closeOnClickOutside = (e) => {
   }
 };
 
-const getIetfLocale = (ietfLocale) => {
-  const nonStandardLocaleMap = { no_NO: 'nb_NO' };
-  return nonStandardLocaleMap[ietfLocale] || ietfLocale;
+const getUniversalNavLocale = () => {
+  const { locale: { prefix: configPrefix = '', ietf = '' } = {} } = getConfig() || {};
```

Comment:
I don't think this is needed, we should be good with `{ locale } = getConfig();`. You can also use locale as an argument, it should help when calling the method since you already have the config object.

Reply:
Made the change 👍